spec/system/task_spec.rb

```diff
@@ -1,24 +1,27 @@
 require 'rails_helper'
+include ApplicationHelper
 
 RSpec.describe 'Task', type: :system do
   describe 'Task一覧' do
+    let(:project) {create(:project)}
+    let(:task) {create(:task, project_id: project.id)}
```

Comment:
`association`が設定されていれば不要な記述になると思います。
```suggestion
    let(:task) { create(:task) }
```